### ISSUE TITLE
Markdown Preview ordered lists broken

### ISSUE THREAD

**Issue by chmielot** [box/box-content-preview#634]
Environment:
- Preview version: https://cdn01.boxcdn.net/platform/preview/1.28.1/en-US/preview.css
- Browser name and version: Chrome Version 63.0.3239.132, but affects all browsers
- Operating system and version: Windows 10 Enterprise 1709

Steps to reproduce the problem:
1. Create a new markdown file with extension `.md`
2. In that markdown file create an ordered list, example:
```markdown
A list of ordered items:

1. List item number one
1. List item number two
1. List item number three
   * Item 4
   * Item 5
```
3. Preview the file in Box

What is the expected behavior? (Screenshots can be helpful here)
It is expected that the ordered list's `list-style-type` is `decimal`. Example:
![image](https://user-images.githubusercontent.com/380087/35920827-44041d22-0c19-11e8-8943-41e4a7b635b9.png)

What went wrong? (Screenshots can be helpful here)
The items of the ordered list are rendered as bullet points:
![image](https://user-images.githubusercontent.com/380087/35920838-534a9fea-0c19-11e8-8ec7-e96f285ad9dc.png)

Link to application or sample code:
The reason for this behaviour seems to be the following entry in the `preview.css` file:
![image](https://user-images.githubusercontent.com/380087/35920921-922bda9e-0c19-11e8-83b0-653f6ab8e002.png)

Can the `list-style-type` be changed to `decimal` for `.bp-text-plain ol` or maybe introduce a new class for the markdown context?

**Comment by DanDeMicco** [comment]
Hey @chmielot thanks for reporting the issue! I looked into the issue and was able to reproduce it. I will be pushing a fix shortly, and it should be available in the next release (1.30.0)

**Comment by DanDeMicco** [comment]
Hey @chmielot no problem! I just did some more testing and it seems to be rendering properly according to github's md format
